drivers/SmartThings/zigbee-switch/profiles/inovelli-vzm31-sn.yml

```diff
@@ -0,0 +1,279 @@
+  Default=25 (2500ms or 2.5s)"
+    required: false
+    preferenceType: number
+    definition:
+      minimum: 0
+      maximum: 126
+      default: 25
+  - name: "parameter2"
+    title: "2. Dimming Speed (Local)"
+    description: "This changes the speed that the light dims up when controlled at the switch. A setting of '0' turns the light immediately on. Increasing the value slows down the transition speed. Value is multiplied by 100ms. 
+  (i.e 25 = 2500ms or 2.5s) Default=127 (Sync with parameter 1)"
+    required: false
+    preferenceType: number
+    definition:
+      minimum: 0
+      maximum: 127
+      default: 127
+  - name: "parameter3"
+    title: "3. Ramp Rate (Remote)"
+    description: "This changes the speed that the light turns on when controlled from the hub. A setting of '0' turns the light immediately on. Increasing the value slows down the transition speed. Value is multiplied by 100ms. 
+  (i.e 25 = 2500ms or 2.5s) Default=127 (Sync with parameter 1)"
+    required: false
+    preferenceType: number
+    definition:
+      minimum: 0
+      maximum: 127
+      default: 127
+  - name: "parameter4"
+    title: "4. Ramp Rate (Local)"
+    description: "This changes the speed that the light turns on when controlled at the switch. A setting of '0' turns the light immediately on. Increasing the value slows down the transition speed. Value is multiplied by 100ms. 
+  (i.e 25 = 2500ms or 2.5s) Default=127 (Sync with parameter 3)"
+    required: false
+    preferenceType: number
+    definition:
+      minimum: 0
+      maximum: 127
+      default: 127
+  - name: "parameter9"
+    title: "9. Minimum Level"
+    description: "The minimum level that the light can be dimmed. Useful when the user has a light that does not turn on or flickers at a lower level."
+    required: false
+    preferenceType: number
+    definition:
+      minimum: 1
+      maximum: 99
+      default: 1
+  - name: "parameter10"
+    title: "10. Maximum Level"
+    description: "The maximum level that the light can be dimmed. Useful when the user wants to limit the maximum brighness."
+    required: false
+    preferenceType: number
+    definition:
+      minimum: 2
+      maximum: 100
+      default: 100
+  - name: "parameter11"
+    title: "11. Invert Switch"
+    description: "Inverts the orientation of the switch. Useful when the switch is installed upside down. Essentially up becomes down and down becomes up."
+    required: false
+    preferenceType: enumeration
+    definition:
+      options:
+        "0": "No (default)"
+        "1": "Yes"
+      default: 0
+  - name: "parameter12"
+    title: "12. Auto Off Timer"
+    description: "Automatically turns the switch off after this many seconds. When the switch is turned on a timer is started. When the timer expires the switch turns off.
+  0=Auto Off Disabled."
+    required: false
+    preferenceType: number
+    definition:
+      minimum: 0
+      maximum: 32767
+      default: 0
```

Comment:
We'd prefer if users used our rules and automations for this sort of behavior.

Reply:
Sure no problem. I can remove it.